### DISCUSSION TITLE
Copilot keeps suggesting code blocks in Markdown

### DISCUSSION THREAD

**Discussion by chylex** [community/community#43589] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi, I find that Copilot is often good at completing plain English sentences, but in one of my Markdown documents it has somehow decided that every single suggestion needs to end by starting a code block:

![obrazek](https://user-images.githubusercontent.com/3685160/211167251-29396f50-70f5-4726-b30d-15525cb2c5ea.png)

I tried regenerating the suggestions a few times, but in 29/30 suggestions it just keeps trying to insert some code block immediately after the sentence. The one suggestion where it didn't do that, it just squeezed in a heading before the inevitable code block.

The same is happening with suggestions directly in the editor:

![obrazek](https://user-images.githubusercontent.com/3685160/211167405-7df16cce-e21b-4a76-b58c-81ed3b142f0b.png)

It can complete sentences or write whole new sentences, but that extra code block makes it quite annoying because I need to delete 2+ new lines every time I accept a suggestion.

Is there a way (or could you implement one) to improve this? Maybe a way to tell Copilot that it should not generate multiple lines, either with a manual toggle or as a rule for certain file types? Or maybe some way to flag suggestions as good or bad? Or something else that could help me steer Copilot in the right direction. Thanks!

Note: [this is the file I was editing](https://github.com/chylex/Apache-Prometheus-Exporter/blob/main/README.md), apart from the beginning there aren't many code blocks, so I'm not sure where it picked up this behavior.

**Comment by historia-studio** [comment]
![image](https://github.com/community/community/assets/145473324/59c16731-7794-483a-b21c-727903c03b46)
I have the same issue, regardless of what language I'm writing in, copilot tries to start a code block at the end of it.

**Comment by maciej-skurznica** [comment]
I have a very similar issue. In every file at the end of the code when I start a new line it suggests ```. It is very annoying. I hope someone has got a solution for this.
<img width="265" alt="Screenshot 2024-04-03 at 20 09 18" src="https://github.com/community/community/assets/83079179/d060b547-ccfc-439d-aa25-91e529bde897">
<img width="343" alt="Screenshot 2024-04-03 at 20 09 24" src="https://github.com/community/community/assets/83079179/eb6d7828-67c3-41a2-8a72-bbee4f5fbca0">
